<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1574</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="TabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1561</width>
      <height>861</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>&amp;Topic</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>&amp;Functionality</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>&amp;Component</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>590</y>
        <width>1171</width>
        <height>221</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>&amp;Component</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>321</width>
          <height>161</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Package</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>&amp;Activities</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Periodic</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>M</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>&amp;Properties</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Domain</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>&amp;Ports</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Topic Type</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Role</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>&amp;Functionalities</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Plugin Interfaces</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_9">
       <attribute name="title">
        <string>&amp;Plugins</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>331</width>
          <height>131</height>
         </rect>
        </property>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <row>
         <property name="text">
          <string/>
         </property>
        </row>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Package</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Type</string>
         </property>
        </column>
       </widget>
      </widget>
     </widget>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>351</width>
        <height>561</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>351</width>
        <height>211</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <row>
       <property name="text">
        <string/>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Property</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Value</string>
       </property>
      </column>
     </widget>
     <widget class="QGraphicsViewEc" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>1171</width>
        <height>561</height>
       </rect>
      </property>
      <property name="dragMode">
       <enum>QGraphicsView::ScrollHandDrag</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>&amp;System</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1574</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Tables</string>
    </property>
    <addaction name="actionAdd_component"/>
    <addaction name="actionAdd_Activity"/>
    <addaction name="actionAdd_Property"/>
    <addaction name="actionAdd_Port"/>
    <addaction name="actionAdd_Functionality"/>
    <addaction name="actionAdd_Plugin"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;?</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Help"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_component">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add Component</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionAdd_Activity">
   <property name="text">
    <string>Add Activity</string>
   </property>
  </action>
  <action name="actionAdd_Property">
   <property name="text">
    <string>Add Property</string>
   </property>
  </action>
  <action name="actionAdd_Port">
   <property name="text">
    <string>Add Port</string>
   </property>
  </action>
  <action name="actionAdd_Functionality">
   <property name="text">
    <string>Add Functionality</string>
   </property>
  </action>
  <action name="actionAdd_Plugin">
   <property name="text">
    <string>Add Plugin</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewEc</class>
   <extends>QGraphicsView</extends>
   <header>QGraphicsViewEc.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
